src/test/java/de/samply/share/client/util/connector/LaplaceMechanismTest.java

```diff
@@ -1,23 +1,46 @@
 package de.samply.share.client.util.connector;
 
+import static java.nio.charset.StandardCharsets.US_ASCII;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import java.io.UnsupportedEncodingException;
-import java.security.NoSuchAlgorithmException;
 import java.security.SecureRandom;
+import org.junit.jupiter.api.RepeatedTest;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvFileSource;
 
 public class LaplaceMechanismTest {
 
+  public static final double EPSILON = 0.28;
+
   @ParameterizedTest
   @CsvFileSource(resources = "LaplaceMechanismTest.csv")
-  public void testGetDisguisedNumber(int number, int result)
-      throws NoSuchAlgorithmException, UnsupportedEncodingException {
+  public void testPrivatize(int number, int result) throws Exception {
     //to get the same randomness for each test invocation
     SecureRandom sr = SecureRandom.getInstance("SHA1PRNG");
-    sr.setSeed("testSeed12".getBytes("us-ascii"));
-    assertEquals(result,
-        LaplaceMechanism.privatize(number, 1, 0.12, sr));
+    sr.setSeed("testSeed12".getBytes(US_ASCII));
+    assertEquals(result, LaplaceMechanism.privatize(number, 1, EPSILON, sr));
+  }
+
+  @RepeatedTest(1000)
```

Comment:
I guess, the `@RepeatedTest` takes action after setting the RNG seed? Otherwise, it would be useless due to the same random values drawn every time.

Reply:
The repeated tests use the variant of privatize, which initialize a new RNG without seed each time.